.github/workflows/main.yaml

```diff
@@ -36,7 +35,6 @@ jobs:
           - "3.10"
           - "3.11"
           - "3.12"
-          - "3.13"
```

Comment:
please don't remove support for 3.13.

in theory pydicom 3 works with 3.13, so maybe you can change the pinning to pydicom>=3.